### ISSUE TITLE
Inde or Index?

### ISSUE THREAD

**Issue by lsl5** [gregcaporaso/An-Introduction-To-Applied-Bioinformatics#194]
Ooooh, sorry so picky.

Chapter 1.1
http://readiab.org/book/latest/1/
<title>inde</title>

I think you want the title to be index? or something more specific?
Thanks,
Linda

**Comment by ebolyen** [comment]
Definitely an issue with our build process, ideally we would have a meaningful title there instead of just `index` and certainly we want something other than `inde`. @gregcaporaso I'll look into this as part of the templating process I'm working on.

**Comment by gregcaporaso** [comment]
Yep, it's also related to a jupyter issue I created a while ago, but can't seem to find it right now. They hard-code the removal of characters from the file name to create the title (so ``index.ipynb`` becomes ``index``, but ``index.md`` becomes ``inde``).

**Comment by lsl5** [comment]
So you would have to manually (script) an edit of all the pages? Eeew.
Linda
<https://www.avast.com/sig-email?utm_medium=email&utm_source=link&utm_campaign=sig-email&utm_content=webmail>
This
email has been sent from a virus-free computer protected by Avast.
www.avast.com
<https://www.avast.com/sig-email?utm_medium=email&utm_source=link&utm_campaign=sig-email&utm_content=webmail>
<#DDB4FAA8-2DD7-40BB-A1B8-4E2AA1F9FDF2>

On Fri, Jan 22, 2016 at 6:32 AM, Greg Caporaso <notifications@github.com>
wrote:

> Yep, it's also related to a jupyter issue I created a while ago, but can't
> seem to find it right now. They hard-code the removal of characters from
> the file name to create the title (so index.ipynb becomes index, but
> index.md becomes inde).
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/gregcaporaso/An-Introduction-To-Applied-Bioinformatics/issues/194#issuecomment-173922669>
> .
>



-- 
Linda Sue Lassiter
lsl5@nau.edu
